### PR TITLE
Status added in Recent Transactions of W token & fixes

### PR DESCRIPTION
**Pull request by giulianoconti**
## Description #805 

- Status added in recent txs of w token.
- Charts styles fixed.
- Show the entire number on Stats.
- Amount renamed to Transfer Count.

status

chartHover

wStats

amountRenamed

### CODE REVIEW

File: src/pages/Analytics/WToken/ByChain.tsx [hunk 1]
```diff
@@ -26,7 +26,10 @@ export const ByChain = ({ activityNotional, activityTx }: ByChainProps) => {
           <div className="bychain-half-table-head">
             <div className="bychain-half-table-head-row">SOURCE CHAIN</div>
             <div className="bychain-half-table-head-row">TARGET CHAIN</div>
-            <div className="bychain-half-table-head-row">AMOUNT</div>
+            <div className="bychain-half-table-head-row">
+              <span className="mobile">TXN COUNT</span>
+              <span className="desktop">TRANSFER COUNT</span>
```
Comment (valentinoConti) [inline src/pages/Analytics/WToken/ByChain.tsx:31]: Shouldn't be "TRANSFERS" ?

File: src/pages/Analytics/WToken/index.tsx [hunk 2]
```diff
@@ -47,31 +57,122 @@ const WToken = () => {
...
+              : "VAA_EMITTED"
+            : isBigTransaction || isDailyLimitExceeded
+            ? "IN_GOVERNORS"
+            : "IN_PROGRESS";
+
+          return {
+            ...tx,
+            STATUS,
+          };
+        });
```
Comment (valentinoConti) [inline src/pages/Analytics/WToken/index.tsx:163]: This is good for now but we need to refactor it later to not have the same logic in multiple places 👍🏼

File: src/pages/Analytics/WToken/ByChain.tsx (hunk 1 above)
Comment (giulianoconti) [reply src/pages/Analytics/WToken/ByChain.tsx:31]: I will check with Sami, but probably yes

File: src/pages/Analytics/WToken/index.tsx (hunk 2 above)
Comment (giulianoconti) [reply src/pages/Analytics/WToken/index.tsx:163]: 📝

File: src/pages/Analytics/WToken/ByChain.tsx (hunk 1 above)
Comment (giulianoconti) [reply src/pages/Analytics/WToken/ByChain.tsx:31]: She told me that "TRANSFER" is ok

**Pull request merged**